client/dist/styles.css

```diff
@@ -45,9 +48,11 @@
   top: 50%;
   transform: translateY(-50%);
 }
+
```

Comment:
I personally don't like the spacing.

Reply:
Yes, let's keep it consistent throughout, I agree re: no spacing. I will adjust on my end before next PR.